tests/publish_subscribe.rs

```diff
@@ -5,9 +5,13 @@ use up_rust::{MockUListener, UMessageBuilder, UTransport, UUri};
 
 mod common;
 
-#[tokio::test(flavor = "multi_thread")]
-#[ignore = "should only be executed with an MQTT broker running"]
+#[tokio::test]
 async fn test_publish_and_subscribe() {
+    env_logger::builder().parse_filters("debug").init();
```

Comment:
can we remove this? If you/we need to have debug output we can simply set RUST_LOGS accordingly, can't we?

Reply:
yes sure there hasn't been any logger init before and I wanted this a a default in case you log it to a file on build machines. And if something fails on CI it is convenient to have logs instead of having to change an action yaml to set RUST_LOG

Reply:
I do not quite see why we would need to debug the tests on the CI machine. If the tests fail, then they should provide relevant information as to what has failed. The debugging, however, should then be done on your development machine, shouldn't it?

Reply:
ok i changed it